requirements.txt

```diff
@@ -1,11 +1,12 @@
 docutils==0.16
+isort==4.3.21
```

Comment:
Better check how we can avoid this pinned version, please

Reply:
Could you review again, please?

Regards.